Formula/p/pivit.rb

```diff
@@ -0,0 +1,24 @@
+class Pivit < Formula
+  desc "Sign and verify data using hardware (Yubikey) backed x509 certificates (PIV)"
```

Comment:
I guess, ssh_key can also be used to sign and verify, correct?
If yes, can we change this to something like
```
sign and verify data using identities (X509 certificate or SSH Keys) stored on hardware (Yubikey)
```